__tests__/interpolate/Interpolate_number_test.re

```diff
@@ -21,7 +32,7 @@ describe("Remap_number", () => {
     it("should handle ranges and domains with non-overlapping inputs", () =>
       Expect.(
         expect(
-          Remap_number.remapf(
+          Interpolate_number.remapf(
```

Comment:
I decided to keep all interpolation related functions in a single module `Interpolate_number`.